cmd/oras/internal/option/spec.go

```diff
@@ -68,21 +67,21 @@ func (is *ImageSpec) Options() string {
 	return strings.Join([]string{
 		ImageSpecV1_1,
 		ImageSpecV1_0,
-		ImageSpecAuto,
 	}, ", ")
 }
 
 // String returns the string representation of the flag.
 func (is *ImageSpec) String() string {
-	return is.Flag
+	// to avoid printing default value in usage doc
+	return ""
 }
 
 // ApplyFlags applies flags to a command flag set.
 func (is *ImageSpec) ApplyFlags(fs *pflag.FlagSet) {
-	// default to auto
+	// default to v1.1, unless --config is used and --artifact-type is not used
 	is.PackVersion = oras.PackManifestVersion1_1
-	is.Flag = ImageSpecAuto
-	fs.Var(is, "image-spec", fmt.Sprintf(`[Experimental] specify manifest type for building artifact. Options: %s (default %q)`, is.Options(), ImageSpecAuto))
+	is.Flag = ImageSpecV1_1
+	fs.Var(is, "image-spec", fmt.Sprintf(`[Experimental] specify manifest type for building artifact. Options: %s (default v1.1, unless --config is used and --artifact-type is not used)`, is.Options()))
```

Comment:
What's the value of `unless` case? It's not clear in the description

Reply:
How about this? `specify manifest type for building artifact. Options: v1.1, v1.0 (default value is v1.1. The default is overridden to v1.0 if --config is specified and --artifact-type is not specified)`

Reply:
Need to optimize the multi-line layout, when this sentence is too long.

Reply:
Looks too long for me @FeynmanZhou to confirm the UX

Reply:
Optimized further with line break: `specify manifest type for building artifact. Options: v1.1(default), v1.0`
`(The default is overridden to v1.0 if --config is specified and --artifact-type is not specified)`

Reply:
I would suggest changing the label of `--image-spec` from [Experimental] to [Preview] and print it out as follows:
 
```
[Preview] specify manifest type for building artifact. Options: v1.1, v1.0 (default v1.1, unless --config is used and --artifact-type is not used)
```

Reply:
@FeynmanZhou The phrasing still uses `unless`, which doesn't clearly described what value has been switched to. Agree on we should change the feature mark from `Experimental` to `Preview`

Reply:
@FeynmanZhou Just realize that we have E2E test for feature marks, changing the mark will cause E2E failure. Let's change the feature mark in another PR.

Reply:
LGTM